sql/core/src/test/resources/sql-tests/inputs/postgreSQL/window_part2.sql

```diff
@@ -15,6 +15,18 @@ CREATE TABLE empsalary (
     enroll_date date
 ) USING parquet;
 
+INSERT INTO empsalary VALUES
+  ('develop', 10, 5200, date '2007-08-01'),
+  ('sales', 1, 5000, date '2006-10-01'),
+  ('personnel', 5, 3500, date '2007-12-10'),
+  ('sales', 4, 4800, date '2007-08-08'),
+  ('personnel', 2, 3900, date '2006-12-23'),
+  ('develop', 7, 4200, date '2008-01-01'),
+  ('develop', 9, 4500, date '2008-01-01'),
+  ('sales', 3, 4800, date '2007-08-01'),
+  ('develop', 8, 6000, date '2006-10-01'),
+  ('develop', 11, 5200, date '2007-08-15');
```

Comment:
Once https://github.com/apache/spark/pull/27439/files#r375009397 is addressed, there's no point of creating this table anymore because it's used nowhere. I think you can comment out both CREATE and INSERT, with commenting it's currently not used.

Reply:
OK